azkaban-exec-server/src/test/java/azkaban/execapp/event/RemoteFlowWatcherTest.java

```diff
@@ -248,7 +296,7 @@ private FlowRunner createFlowRunner(final File workingDir, final ExecutorLoader
       final EventCollectorListener eventCollector, final String flowName, final int execId,
       final FlowWatcher watcher, final Integer pipeline, final Props azkabanProps)
       throws Exception {
-    final File testDir = new File("unit/executions/exectest1");
+    final File testDir = new File("../test/src/test/resources/azkaban/test/executions/exectest1");
```

Comment:
Please use the wrapper method to get the file object.

Reply:
I think this is already fixed in the pre-requisite PR that enables the RemoteFlowWatcherTest. Once that is merged I will update this PR.